# WifiScanList Message

std_msgs/Header header

openpilot/WifiScan[] wifiScan
